### PR TITLE
how-to added

### LINKED ISSUE #109: create CONTRIBUTING.md file
**What would you like to be added**:
General guidelines 

`sub-topics:`
- Before you get started
- How to contribute
- Setting up local environment 

**Why is this needed**:
Helps new contributors to understand workflow of the project

@RainbowMango @Arhell What other sub-topics should we include?

### PR DESCRIPTION
**Pull request by rgrupesh**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Enhances documentation 

**Which issue(s) this PR fixes**:
Fixes #109

**Special notes for your reviewer**:
Should we include additional information about contributing, such as `coding style` and `instructions for setting up locally`, on the same page as this one or on a different page?
@RainbowMango

### CODE REVIEW

File: docs/contributor/contribute-docs.md [hunk 1]
```diff
@@ -16,6 +20,27 @@ the `karmada-io/website` repository.
...
+
+1. Check the [issue tracker](https://github.com/karmada-io/karmada/issues) for any open issues related to the feature or bug you want to work on. If there isn't an issue for it, feel free to create one.
+
+2. Assign yourself to the issue you want to work on.
+
+3. Fork the repository.
+
+4. Create a new branch with a descriptive name for the issue you are working on.
+
+5. Write your code/chnages and commit your changes.
```
Comment (RainbowMango) [inline docs/contributor/contribute-docs.md:36]: typo `chnages`

File: docs/contributor/contribute-docs.md [hunk 2]
```diff
@@ -16,6 +20,27 @@ the `karmada-io/website` repository.
 - For our docs we use markdown. If you are unfamiliar with Markdown, please see https://guides.github.com/features/mastering-markdown/ or https://www.markdownguide.org/ if you are looking for something more substantial. 
 - We get some additions through [Docusaurus 2](https://docusaurus.io/), a model static website generator.
 
+
+## How to Contribute
+
+If you want to contribute to Karmada, you can follow these steps:
```
Comment (RainbowMango) [inline docs/contributor/contribute-docs.md:26]: The current document focuses on `docs` contributing, but the added thing seems like [the common process](https://github.com/karmada-io/karmada/blob/master/CONTRIBUTING.md).

I don't think we should do it here.

Comment (rgrupesh) [reply docs/contributor/contribute-docs.md]: Removed How-to section from here

**Comment by RainbowMango** [comment]
Hi @rgrupesh merge commit is not allowed, please rebase.
Use git rebase to reapply your commits on top of the target branch. Detailed instructions for doing so can be found [here](https://git.k8s.io/community/contributors/guide/github-workflow.md#4-keep-your-branch-in-sync).

**Comment by RainbowMango** [comment]
Please help [squash commits](https://karmada.io/docs/contributor/github-workflow/#squash-commits), then will be all set, thanks.

**Comment by RainbowMango** [comment]
/hold
@rgrupesh do you need help?

**Comment by rgrupesh** [comment]
> /hold @rgrupesh do you need help?

@RainbowMango I squashed my first commit to this branch with the previous commit of the main branch, and left newer commit `how-to removed and markdown synatx link added` as it is.

**Comment by rgrupesh** [comment]
> > /hold @rgrupesh do you need help?
> 
> @RainbowMango I squashed my first commit to this branch with the previous commit of the main branch, and left newer commit `how-to removed and markdown synatx link added` as it is.

@RainbowMango

**Comment by RainbowMango** [comment]
I don't think this PR should contain other guys' commit. Please rebase your branch and push again.

**Pull request closed without merging**